WMGM

```diff
@@ -1,5 +1,20 @@
 #!/usr/bin/env python
 
+##############################################################
+#
+# This script is initiated by a daemon that communicates with the data submission on NiftyWeb.
+#
+# Two NIfTI files are required: an initial scan (image 1) and a re-scan without repositioning (image 2).
+#
+# It is executed in the following way:
+#
+#    python PATH_OF_THIS_SCRIPT/WMGM <data ID> <image 1> <image 2>
```

Comment:
the version of python must be indicated-- how it is explained in the other headers

Reply:
It doesn't require SCT python like the others. Is that what you mean?

Reply:
- Bob buys a computer, installs a new OS without python, clone your repos, runs it --> it crashes because "python" is not recognized.  
- Robert installs python via conda, clones your repos, runs it --> it crashes because "pandas" is not a default module of conda.

Conclusion: ALWAYS defines a python with dependencies. Simple solution: suggests the SCT python because we know it is going to work.

Reply:
Ah, I see what you're saying.